Mathlib/Analysis/SpecialFunctions/Log/Base.lean

```diff
@@ -397,7 +397,47 @@ theorem logb_eq_zero : logb b x = 0 ↔ b = 0 ∨ b = 1 ∨ b = -1 ∨ x = 0 ∨
   simp_rw [logb, div_eq_zero_iff, log_eq_zero]
   tauto
 
--- TODO add other limits and continuous API lemmas analogous to those in Log.lean
+theorem tendsto_logb_nhdsWithin_zero (hb : 1 < b) :
+    Tendsto (logb b) (𝓝[≠] 0) atBot :=
+  tendsto_log_nhdsWithin_zero.atBot_div_const (log_pos hb)
+
+theorem tendsto_logb_nhdsWithin_zero_of_base_lt_one (hb₀ : 0 < b) (hb : b < 1) :
+    Tendsto (logb b) (𝓝[≠] 0) atTop :=
+  tendsto_log_nhdsWithin_zero.atBot_mul_const_of_neg (inv_lt_zero.2 (log_neg hb₀ hb))
+
+lemma tendsto_logb_nhdsWithin_zero_right (hb : 1 < b) : Tendsto (logb b) (𝓝[>] 0) atBot :=
+  tendsto_log_nhdsWithin_zero_right.atBot_div_const (log_pos hb)
+
+lemma tendsto_logb_nhdsWithin_zero_right_of_base_lt_one (hb₀ : 0 < b) (hb : b < 1) :
+    Tendsto (logb b) (𝓝[>] 0) atTop :=
+  tendsto_log_nhdsWithin_zero_right.atBot_mul_const_of_neg (inv_lt_zero.2 (log_neg hb₀ hb))
+
+theorem continuousOn_logb : ContinuousOn (logb b) {0}ᶜ := continuousOn_log.div_const _
+
+/-- The real logarithm base b is continuous as a function from nonzero reals. -/
+@[continuity]
+theorem continuous_logb : Continuous fun x : { x : ℝ // x ≠ 0 } => logb b x :=
+  continuous_log.div_const _
+
+/-- The real logarithm base b is continuous as a function from positive reals. -/
+@[continuity]
+theorem continuous_logb' : Continuous fun x : { x : ℝ // 0 < x } => logb b x :=
+  continuous_log'.div_const _
+
+theorem continuousAt_logb (hx : x ≠ 0) : ContinuousAt (logb b) x :=
+  (continuousAt_log hx).div_const _
```

Comment:
If you write this in compositional form this would be a nice `fun_prop` lemma. That is:
```lean
theorem ContinuousAt.logb {f : α → ℝ} {x : α} (hf : ContinuousAt f x) (hx : f x ≠ 0) :
    ContinuousAt (fun y ↦ logb b (f y)) x :=
  sorry
```
Of course, this probably also applies to `continuousAt_log`

Reply:
This is one of the lemmas I haven't transferred yet in this PR, it'll be in the next PR in this series. `ContinuousAt.log` exists, and is a `fun_prop` lemma, but for ease of review I decided to separate it out.